pkg/cmd/builder/build.go

```diff
@@ -235,19 +235,19 @@ func generateBuildctlArgs(ctx context.Context, client *containerd.Client, option
 	}
 	if tags = strutil.DedupeStrSlice(options.Tag); len(tags) > 0 {
 		ref := tags[0]
-		named, err := distributionref.ParseNormalizedNamed(ref)
+		parsedReference, err := referenceutil.Parse(ref)
```

Comment:
Harmonizing variables names from `named`, `canonicalRef`, etc, to just the same `parsedReference`.